cloudfront/wellcomelibrary.org/terraform/cert.tf

```diff
@@ -0,0 +1,20 @@
+module "cert" {
+  source = "../../modules/certificate"
+
+  domain_name = "wellcomelibrary.org"
+
+  zone_id = data.aws_route53_zone.zone.id
+
+  providers = {
+    aws     = aws.us_east_1
+    aws.dns = aws.dns
+  }
+
+  subject_alternative_names = ["*.wellcomelibrary.org"]
```

Comment:
Do we want to do this and not have a cert per subdomain? I keep toing and froing about having a cert for each subdomain, which does feel less couple-y - and think I prefer having a cert per sub-domain.

Reply:
Fewer certs makes sense I think, less things that require renewing. I'm not sure. We can always add more certs later.